java-agent/src/nativeMain/kotlin/com/epam/drill/agent/Agent.kt

```diff
@@ -87,11 +88,18 @@ object Agent {
 
     private val pathSeparator = if (Platform.osFamily == OsFamily.WINDOWS) "\\" else "/"
 
+    private fun getDrillInstallationDir() = open("/proc/self/cmdline", O_RDONLY).let(::Input).readText().run {
+        val isContainsAgetPath: (String) -> Boolean = { it.contains("-agentpath:") }
```

Comment:
fix the typo `isContainsAgetPath` - `isContainsAgentPath`

Reply:
Fixed